sentinel-adapter/sentinel-apache-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/DubboUtils.java

```diff
@@ -57,20 +57,20 @@ public static String getResourceName(Invoker<?> invoker, Invocation invocation,
         return buf.toString();
     }
 
-    public static String getResourceName(Invoker<?> invoker, Invocation invocation, String prefix) {
+    public static String getMethodResourceName(Invoker<?> invoker, Invocation invocation, String prefix) {
         if (StringUtil.isNotBlank(prefix)) {
             return new StringBuilder(64)
                     .append(prefix)
-                    .append(getResourceName(invoker, invocation, DubboConfig.getDubboInterfaceGroupAndVersionEnabled()))
+                    .append(getMethodResourceName(invoker, invocation, DubboAdapterGlobalConfig.getDubboInterfaceGroupAndVersionEnabled()))
                     .toString();
         } else {
-            return getResourceName(invoker, invocation, DubboConfig.getDubboInterfaceGroupAndVersionEnabled());
+            return getMethodResourceName(invoker, invocation, DubboAdapterGlobalConfig.getDubboInterfaceGroupAndVersionEnabled());
         }
     }
```

Comment:
`getMethodResourceName` has prefix configuration, maybe getInterfaceResourceName can be added, also can refer to PR of dubbo-adaptor of 2.6.

Reply:
Ok, this piece is really my incomplete consideration, thank you for correcting, and it has been fixed

Reply:
So modest, very nice work! 👍